### PR TITLE
Support large tensors in grouped conv fwd

### PR DESCRIPTION
**Pull request by bartekxk**

### CODE REVIEW

**Review by geyyer — APPROVED** [review]
LGTM!

**Review by zjing14 — APPROVED** [review]

File: include/ck/tensor_operation/gpu/grid/gridwise_gemm_multiple_d_wmma_cshuffle.hpp [hunk 1]
```diff
@@ -60,12 +60,9 @@ __global__ void
         __builtin_amdgcn_readfirstlane(get_grid_size() / batch_count);
     const index_t g_idx = __builtin_amdgcn_readfirstlane(get_block_1d_id() / num_blocks_per_batch);
 
-    const long_index_t a_batch_offset = __builtin_amdgcn_readfirstlane(
```
Comment (zjing14) [inline include/ck/tensor_operation/gpu/grid/gridwise_gemm_multiple_d_wmma_cshuffle.hpp:63]: Why this change?
